test/DynamoCoreTests/UnitsOfMeasureTests.cs

```diff
@@ -766,6 +766,7 @@ internal class ForgeUnitsTests : UnitTestBase
         [Test, Category("UnitTests")]
         public void CanCreateForgeUnitType_FromLoadedTypeString()
         {
+            // Exact version exists: use the version as-specified
```

Comment:
The test environment now includes `2.0.0` schemas, so the test cases were updated accordingly. For version-based tests, we replaced `1.0.2` with `99.9.9` to represent a "future version," since `99.9.9` is guaranteed not to exist and thus serves as a reliable placeholder. For the same reason, we use `0.0.1` as the "past version," ensuring it also doesn’t exist.